coding_style_and_standards.md

```diff
@@ -33,6 +34,19 @@ maintained by the ManageIQ team.
   in the [.rubocop_base.yml](.rubocop_base.yml) file, which is inherited by most
   projects in the ManageIQ organization.
 
+## Documentation
+
+We chose [yardoc](https://yardoc.org/) to create inline code documentation. For now documentation is scarce but
+we hope the situation will improve. This is why we limit the created documentation to those files that are well
```

Comment:
s/chose/use/

s/we hope the situation will improve/we encourage contributors to add it whenever possible/

s/This is why we limit/Until more of the codebase is documented, we limit/